cmd/pj-rehearse/main.go

```diff
@@ -147,7 +150,11 @@ func main() {
 		gracefulExit(o.noFail, misconfigurationOutput)
 	}
 
-	changedPresubmits := diffs.GetChangedPresubmits(prowConfig, prowPRConfig, logger)
+	changedPresubmits := diffs.GetChangedPresubmits(prowConfig, prowPRConfig, logger, o.allowVolumes)
+	if len(changedPresubmits) == 0 {
+		logger.Info("No jobs have been found to rehearse")
```

Comment:
Why?

Reply:
if there are no jobs found to rehearse, there is no reason to continue, right?

Reply:
But we're handling it already, and we even have tests to that effect. Why change it?